### ISSUE TITLE
Cannot install reactive_freight

### ISSUE THREAD

**Issue by mensfeld** [brodyhoskins/reactive_freight#5]
when:

```
gem install reactive_freight
```

then:

```
ERROR:  Could not find a valid gem 'reactive_shipping' (~> 3.0.1) (required by 'reactive_freight' (>= 0)) in any repository
```

there's a deps mismatch as the shipping is `3.0.0`.

Have a great day.

**Comment by brodyhoskins** [comment]
This has been resolved in 3932fd3, thank you.

**Comment by mensfeld** [comment]
Thanks. Closing and looking forward to RG release.
